api/src/main/java/io/strimzi/api/kafka/model/KafkaSpec.java

```diff
@@ -113,6 +114,17 @@ public void setClientsCa(CertificateAuthority clientsCa) {
         return maintenanceTimeWindows;
     }
 
+    @Description("Configuration for JmxTrans. When the key is present a JmxTransSpec deployment is created for gathering JMX metrics from each Kafka broker. " +
+            "For more information, go to https://github.com/jmxtrans/jmxtrans[JmxTrans GitHub]")
+    @JsonProperty("jmxTrans")
```

Comment:
Any reason why not name the property `jmsTrans` directly? It would make it more readable since you would know what to use int he CR without searching for the annotations.